lib/runtime/src/metrics.rs

```diff
@@ -1226,37 +1220,42 @@ dynamo_component_testintcounter{{dynamo_namespace="ns345"}} 12345"#
         let filtered_drt_output =
             super::test_helpers::remove_nats_lines(&drt_output_raw).join("\n");
 
-        let expected_drt_output = format!(
-            r#"# HELP dynamo_component_testcounter A test counter
+        let expected_drt_output = r#"# HELP dynamo_component_testcounter A test counter
```

Comment:
Do we need a new test case that would've caught the blocking/failing behavior before this fix?